src/main/kotlin/com/clouway/bankapp/adapter/gae/JsonEntityHelpers.kt

```diff
@@ -0,0 +1,56 @@
+package com.clouway.bankapp.adapter.gae
+
+import com.clouway.bankapp.core.*
+import com.google.appengine.api.datastore.DatastoreService
+import com.google.appengine.api.datastore.Entity
+import com.google.appengine.api.datastore.EntityNotFoundException
+import com.google.appengine.api.datastore.Key
+import com.google.appengine.api.memcache.MemcacheService
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.util.*
+
+/**
+ * @author Tsvetozar Bonev (tsbonev@gmail.com)
+ */
+fun <T> MemcacheService.get(key: String, typeOfT: Class<T>, transformer: JsonTransformerWrapper): Optional<T> {
+    val entity = this.get(key) ?: return Optional.empty()
+    return Optional.of(transformer.fromJson(entity as String, typeOfT))
+}
+
+fun <T> DatastoreService.get(key: Key, typeOfT: Class<T>, transformer: JsonTransformerWrapper): Optional<T> {
```

Comment:
JsonTransformerWrapper  could be <I> JsonSerializer

Reply:
Renadmed class to JsonSerializer and removed it from the datastore package